sensor_msgs/LaserScan laser_scan
---
nav_msgs/OccupancyGrid laser_image